<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScreenshotMaker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="captureFullScreenButton">
        <property name="toolTip">
         <string>Сделать снимок всего экрана</string>
        </property>
        <property name="text">
         <string>Весь экран</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="captureAreaButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Выделить область для снимка</string>
        </property>
        <property name="text">
         <string>Выделить область</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Сохранить скриншот в файл</string>
        </property>
        <property name="text">
         <string>Сохранить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="toolTip">
         <string>Закрыть приложение</string>
        </property>
        <property name="text">
         <string>Выход</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="screenshotLabel">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    border: 2px solid #cccccc;
    background-color: #f0f0f0;
    padding: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>Размер: 0×0</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="copyToClipboardButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Копировать в буфер</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
